project.properties

```diff
@@ -0,0 +1,7 @@
+dafnyVersion=4.9.0
+dafnyVerifyVersion=4.9.0
+dafnyFormatVersion=4.9.0
```

Comment:
In contrast to the other variable and workflow names,
these are perfect!

Apply this naming scheme above!